tools/shared.py

```diff
@@ -261,10 +261,24 @@ def generate_config(path, first_time=False):
 # The search order from the config file is as follows:
 # 1. Specified on the command line (--em-config)
 # 2. Specified via EM_CONFIG environment variable
-# 3. If emscripten-local .emscripten file is found, use that
-# 4. Fall back users home directory (~/.emscripten).
+# 3. Local .emscripten file, if found
+# 4. Local .emscripten file, as written by emsdk --embedded (two levels up), if found.
```

Comment:
It says "two levels" here, but without a little graph like in that comment of yours, it's very unclear I think.

likewise, on line 269 it also says "two levels" but doesn't explain.

If I'm being unclear here, I apologize, and feel free to land this - I'll open a followup PR.